packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -612,6 +612,11 @@ class _GlowingOverscrollIndicatorPainter extends CustomPainter {
   }
 }
 
+enum _StretchDirection {
+  down,
+  up
```

Comment:
I'm not entirely sure about the naming here (`up` and `down`), since this implies verticality, even though it applies to vertical **and** horizontal scrolling. Any suggestions?

Reply:
Typically we use terms like leading and trailing so it can describe both vertical and horizontal cases.